ntoskrnl/mm/section.c

```diff
@@ -1599,31 +1592,34 @@ MmNotPresentFaultSectionView(PMMSUPPORT AddressSpace,
     /* See if we should use a private page */
     if (HasSwapEntry)
     {
-        SWAPENTRY DummyEntry;
-
-        MmGetPageFileMapping(Process, Address, &SwapEntry);
+        MmDeletePageFileMapping(Process, Address, &SwapEntry);
         if (SwapEntry == MM_WAIT_ENTRY)
         {
-            MmUnlockAddressSpace(AddressSpace);
-            YieldProcessor();
-            MmLockAddressSpace(AddressSpace);
+            /* Someone else is handling this page fault.
+             * Restore this. */
+            MmCreatePageFileMapping(Process, Address, MM_WAIT_ENTRY);
+            /* Let the other one a chance to finish */
```

Comment:
```suggestion
            /* Let the other one have a chance to finish */
```

Reply:
https://www.linguee.fr/anglais-francais/traduction/give+a+chance.html ;-)

Reply:
ok I was perturbed due to the "let"